cmd/pebble/main.go

```diff
@@ -33,6 +36,10 @@ func main() {
 		"strict",
 		false,
 		"Enable strict mode to test upcoming API breaking changes")
+	resolverAddress := flag.String(
```

Comment:
I think its probably best to have this as only a config parameter instead of both a config parameter and a command line flag.

Reply:
The command line flag is more easy to use in a container context: create a file with a dynamic IP is not really easy to do.

Reply:
OK. I don't have strong preferences about whether it be a command line flag or a config parameter, just that it not be both :-)

If the command line flag is easier for the container context can you remove the config parameter?

Reply:
it is done :smiley: